### ISSUE TITLE
No info about how to setup mysql database

### ISSUE THREAD

**Issue by gerchicov-bp** [enjin/Enjin-Coin-PHP-SDK#1]
I run `composer run test` and get a lot of errors linked with mysql. Should I perform additional actions with mysql database you didn't specify in readme? Or will it work after official release only?

**Comment by coinfork** [comment]
You're correct, there are some existing errors. The SDK is still pre-alpha so we are developing core features on the master branch that are in various states of completion. We're not worrying about all unit tests passing yet until the SDK is in alpha, which shouldn't be far off, in 1-2 weeks.
